### PR TITLE
#831 - Added a check to avoid NullPointerException.

### PR DESCRIPTION
**Pull request by mdaliejaz**

### CODE REVIEW

File: plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java [hunk 1]
```diff
@@ -18,8 +18,8 @@
 
 import com.thoughtworks.go.plugin.access.config.PluginPreferenceStore;
 
-public final class PluggableTaskConfigStore extends PluginPreferenceStore<TaskPreference> {
```
Comment (pshivana) [inline plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java]: Any reason why final keyword is removed here?

**Comment by arvindsv** [comment]
My guess is that the class is being mocked in the TaskViewServiceTest. Can't mock when it is final. Classes shouldn't be final, IMHO.

File: plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java (hunk 1 above)
Comment (arvindsv) [inline plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java]: My guess is that the class is being mocked in the TaskViewServiceTest. Can't mock when it is final. Classes shouldn't be final, IMHO.

Comment (pshivana) [inline plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java]: Alright

Comment (mdaliejaz) [inline plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/pluggabletask/PluggableTaskConfigStore.java]: As @arvindsv mentioned, I needed to mock and hence had to remove final keyword.
Sorry for not replying earlier.

**Pull request merged**